dev/sparktestsupport/modules.py

```diff
@@ -15,12 +15,14 @@
 # limitations under the License.
 #
 
+from functools import total_ordering
```

Comment:
This is only available in Python 2.7+, but that might be fine: AMPLab Jenkins runs a compatible Python version and I figure that people have control over their dev environments and can upgrade if necessary.